synapse/replication/http/membership.py

```diff
@@ -181,9 +181,9 @@ async def _serialize_payload(room_id, user_id, change):
         Args:
             room_id (str)
             user_id (str)
-            change (str): Either "joined" or "left"
+            change (str): "left"
         """
-        assert change in ("joined", "left")
+        assert change == "left"
```

Comment:
Would it make more sense to update `ReplicationUserJoinedLeftRoomRestServlet` to be `ReplicationUserLeftRoomRestServlet`?

Reply:
shrugface. Maybe. Probably better to do some more complete refactoring to sort out the `user_left_room` case.